default/specials.txt

```diff
@@ -339,7 +339,7 @@ Special
             scope = Source
             activation = OwnedBy affiliation = AnyEmpire
             effects = [
-                SetDetection value = Value + 75
+                SetDetection value = Value + Min(Target.Construction / 30, 1) * 75
```

Comment:
Actually, I think you got that a bit wrong - if I'm not mistaken, that formula means 75 is the max detection range you get, for which you need infra 30. Below infra 30, the detection range bonus is reduced accordingly - if you have infra 15 (half of 30), you get only half of the bonus (37.5 in this case). And so on.

I definitely prefer it this way. Detection ranges are far too high IMO already.